enzy_htp/_interface/pymol_interface.py

```diff
@@ -177,6 +179,21 @@ def load_enzy_htp_stru(self, session: pymol2.PyMOL, stru: Structure) -> Tuple[st
 
 
         return (pymol_obj_name, session)
+    
+    def load_enzy_htp_stru_esm(self, session: pymol2.PyMOL, stru_esm: StructureEnsemble) -> Tuple[str, pymol2.PyMOL]:
+        """Convert enzy_htp.structure.StructureEnsemble object into a pymol object in a pymol2.PyMOL() session.
+        
+        Args:
+            session (pymol2.PyMOL): Current pymol2.PyMOL() session instance.
+            stru_esm (StructureEnsemble): A StructureEnsemble instance.
+
+        Returns:
+            pymol_obj_name (str): The name of the loaded PyMOL object.
+            session (pymol2.PyMOL): Current pymol2.PyMOL() session instance.
+        """
+        self.check_pymol2_installed()
+        prmtop_str = stru_esm._topology
+        pass
```

Comment:
What is this? If this is no longer needed in this PR but potentially valuable for future development, please add an Exception that says it is unfinished.